bundles/org.openhab.binding.digiplex/src/main/java/org/openhab/binding/digiplex/internal/communication/events/TroubleEvent.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.digiplex.internal.communication.events;
+
+import org.openhab.binding.digiplex.internal.communication.DigiplexMessageHandler;
+
+/**
+ * Message providing global trouble status
+ *
+ * @author Robert Michalak - Initial contribution
+ *
+ */
+public class TroubleEvent extends AbstractEvent {
```

Comment:
Please add `@NonNullByDefault` to all of your classes

Reply:
I filled the gaps in all classes, not only those strictly related to this new feature. This is why I put them in a separate commit. Not that I think it makes sense in all cases, as some of them contain only primitive types, but rules are rules... ;)